jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java

```diff
@@ -97,8 +97,10 @@ public void setMaskPropagationEnabled(boolean maskPropagationEnabled) {
     }
 
     /**
-     * Applies an animation mask to this action, if the {@link BaseAction#maskPropagationEnabled} is 
-     * enabled, the animation mask is propagated to the child actions (or wrapped actions).
+     * Internal use only.
+     *
+     * Dispatched from {@link com.jme3.anim.AnimLayer#update(float)}. Applies an animation mask to this action, 
+     * if the {@link BaseAction#maskPropagationEnabled} is enabled, the animation mask is propagated to the child actions (or wrapped actions).
      * 
```

Comment:
I marked this as an internal use-only method, and clarified it's being called from the `AnimLayer#update(...)`.